libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Recognizers/EntityRecognizers/Microsoft.CurrencyEntityRecognizer.schema

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/master/schemas/component.schema",
-    "$role": "implements(Microsoft.EntityRecognizers)",
+    "$role": "implements(Microsoft.IEntityRecognizer)",
     "title": "Currency Entity Recognizer",
     "description": "Recognizer which recognizes currency.",
```

Comment:
>"Recognizer which recognizes currency.", [](start = 19, length = 40)

"Currency recognizer" and we are inconsistent about period or not.

Reply:
I would rename all of these, "Recognizer which recognizes X" is just wordy and hides the most important part, i.e. X.

---
In reply to: [395805747](https://github.com/microsoft/botbuilder-dotnet/pull/3602#discussion_r395805747) [](ancestors = 395805747)